src/roiextractors/segmentationextractor.py

```diff
@@ -285,7 +289,7 @@ def set_times(self, times: ArrayType):
         assert len(times) == self.get_num_frames(), "'times' should have the same length of the number of frames!"
         self._times = np.array(times, dtype=np.float64)
 
-    def frame_to_time(self, frame_indices: Union[IntType, ArrayType]) -> Union[FloatType, ArrayType]:
+    def frame_to_time(self, frames: Union[IntType, ArrayType]) -> Union[FloatType, ArrayType]:
```

Comment:
We are calling `frame_idxs` in the imaging extractor, we should make this consistent across the library I think.

Reply:
`frames` is the general argument call passed to `frame_to_time` in the ImagingExtractor: https://github.com/catalystneuro/roiextractors/blob/master/src/roiextractors/imagingextractor.py#L69

So I'm being consistent with that.

As for `get_frames` taking `frame_idxs`, yeah that should probably be standardized

Reply:
Yep. Makes sense:
https://github.com/catalystneuro/roiextractors/issues/202